drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -29,4 +29,16 @@ config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
 	  that API.  By default the page size corresponds to the block
 	  size (65536).  Other option include the sector size (4096).
 
+config NORDIC_QSPI_NOR_FLASH_ALLOW_STACK_USAGE_FOR_DATA_IN_FLASH
+	bool "Use stack for writing data stored in flash"
+	help
+	  When this option is enabled writing data from FLASH is possible.
+	  However the driver will use stack to copy the data from flash onto
+	  the stack then start the DMA transfer. If you get an -EINVALID error
```

Comment:
Thanks for adding. If you have the time, it would be better to have this documentation in the header file for qspi_nor_write, since I think that's where the user will look if they get the error, but it's not a must.

Reply:
Okei I'll try to move the information there.